pkg/runner/runner.go

```diff
@@ -6,10 +6,9 @@ import (
 
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
-	"github.com/NethermindEth/cairo-vm-go/pkg/parsers/starknet"
 	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
 	"github.com/NethermindEth/cairo-vm-go/pkg/vm"
-	"github.com/NethermindEth/cairo-vm-go/pkg/vm/builtins"
+	builtins "github.com/NethermindEth/cairo-vm-go/pkg/vm/builtins"
```

Comment:
No need for an alias here, right? You can already refer to the package as `builtins` by default. You would only use an alias if there was a need to resolve a conflict or if you wanted to use a different name, but neither is the case here.

Reply:
Done! ✅